tests/zfs-tests/tests/functional/checksum/filetest_001_pos.ksh

```diff
@@ -86,7 +90,7 @@ log_must zpool import $TESTPOOL
 log_must zpool scrub $TESTPOOL
 log_must wait_scrubbed $TESTPOOL
 
-zpool status -P -v $TESTPOOL | grep $firstvdev | read -r name state rd wr cksum
+cksum=$(zpool status -P -v testpool | grep "$firstvdev" | awk '{print $5}')
```

Comment:
Maybe we should use the variable "$TESTPOOL" instead of hardcoding "testpool" here.

Reply:
Thank you for catching that.